test/verify/check-storage-stratis

```diff
@@ -0,0 +1,370 @@
+#
+# Copyright (C) 2021 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from packagelib import *
+from storagelib import *
+from testlib import *
+
+
+@skipImage("No Stratis", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-2004")
+class TestStorageStratis(StorageCase):
+
+    def testBasic(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        dev_1 = "/dev/sda"
+        m.add_disk("4G", serial="DISK1")
+        b.wait_in_text("#drives", dev_1)
+
+        dev_2 = "/dev/sdb"
+        m.add_disk("4G", serial="DISK2")
+        b.wait_in_text("#drives", dev_2)
+
+        dev_3 = "/dev/sdc"
+        m.add_disk("4G", serial="DISK3")
+        b.wait_in_text("#drives", dev_3)
+
+        dev_4 = "/dev/sdd"
+        m.add_disk("4G", serial="DISK4")
+        b.wait_in_text("#drives", dev_4)
+
+        # Create a pool
+        self.dialog_open_with_retry(trigger=lambda: self.devices_dropdown("Create Stratis pool"),
+                                    expect=lambda: (self.dialog_is_present('disks', dev_1) and
+                                                    self.dialog_is_present('disks', dev_2) and
+                                                    self.dialog_check({"name": "pool0"})))
+        self.dialog_set_val("disks", {dev_1: True, dev_2: True})
+        b.assert_pixels("#dialog", "create-pool")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text("#devices", "pool0")
+        b.assert_pixels("#devices", "pool-row")
+
+        # Check that the next name is "pool1"
+        self.devices_dropdown("Create Stratis pool")
+        self.dialog_wait_open()
+        self.dialog_wait_val("name", "pool1")
+        self.dialog_cancel()
+        self.dialog_wait_close()
+
+        b.click('.sidepanel-row:contains("pool0")')
+        b.wait_visible('#storage-detail')
+
+        # Create two filesystems
+        b.click("button:contains(Create new Filesystem)")
+        self.dialog_wait_open()
+        self.dialog_set_val('name', 'fsys1')
+        self.dialog_set_val('mount_point', '/run/fsys1')
+        b.assert_pixels("#dialog", "create-fsys")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text("#detail-content", "fsys1")
+        b.assert_pixels("#detail-content", "fsys-row")
```

Comment:
Can you please also check that the file system is actually mounted? calling `mountpoint` or `findmnt`, and creating a file in it to make sure it's writable.

Reply:
Done